sencha-workspace/SlateAdmin/app/controller/People.js

```diff
@@ -253,7 +255,7 @@ Ext.define('SlateAdmin.controller.People', {
                 }
 
                 me.selectPerson(person, function() {
-                    ExtHistory.resumeState();
+                    ExtHistory.resumeState(false);
```

Comment:
@chris adding false corrected an issue where after loading my people/all path it would switch the url back to people.  It did not seem to affect any other links nor paths, but you might have a better idea of what was causing the issue